modules/light-clients/08-wasm/module.go

```diff
@@ -114,7 +120,11 @@ func (AppModule) ProposalMsgs(simState module.SimulationState) []simtypes.Weight
 	return simulation.ProposalMsgs()
 }
 
-func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, bz json.RawMessage) []abci.ValidatorUpdate {
+func (AppModule) ValidateGenesis(codec.JSONCodec, client.TxEncodingConfig, json.RawMessage) error {
+	return nil
+}
```

Comment:
this comes from HasGenesis interface?

Reply:
that's right, from the embedded `HasGenesisBasics` interface